slack/plugin.py

```diff
@@ -6,10 +6,123 @@
 import httpx
 
 from lutraai.augmented_request_client import AugmentedTransport
-from lutraai.decorator import purpose
+from lutraai.decorator import purpose, inject
+from lutraai.injectables import AuthenticatedClient
 
 _BOT_NAME = "Lutra"
 
+BOT_CLIENT = AuthenticatedClient(
```

Comment:
Note: For now, you'll just construct these injectable things inline directly in your code.

Later we could hide them from the code and render them as UI, or get them from somewhere else, or whatever we want.